server/mvcc/backend/tx_buffer.go

```diff
@@ -37,11 +37,12 @@ func (txb *txBuffer) reset() {
 // txWriteBuffer buffers writes of pending updates that have not yet committed.
 type txWriteBuffer struct {
 	txBuffer
-	seq bool
+	seq map[string]bool
```

Comment:
Please comment about semantic of this field (what's the key and what the value represents).